src/main/java/seedu/tracker/model/module/Title.java

```diff
@@ -10,7 +10,7 @@
 public class Title {
 
     public static final String MESSAGE_CONSTRAINTS = "Title can take any values, but the first character should "
-            + "be alphanumeric. And Title should not be blank";
+            + "be alphanumeric. It should not be blank or more than 60 characters";
```

Comment:
I think we would have to update our UG to reflect these changes. No worries, i'll add them in later